### PR TITLE
Add the Arabic translation (1.0.0)

### PR DESCRIPTION
**Pull request by RIYADHABBES**
Hello Olivier,

I hope you're doing fine.

I found your changelog website while navigating on the net.
I didn't find Arabic language and since your project is open for contribution, I thought to add the corresponding translation.
Also, I haven't Arabic native speakers in my network so I couldn't add some reviewers.

My thanks to you and all the contributors
best wishes

### CODE REVIEW

**Comment by olivierlacan** [comment]
Hi @RIYADHABBES, thank you so much for working on this! I'll ask on Mastodon if some Arabic speakers could review this before we merge. 

I really appreciate your effort! 😃

**Review by m-soli — COMMENTED** [review]
maybe use the similar style changes like the ones on the farsi version to align content right and force period at end of sentences to the left.

File: source/ar/1.0.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+  %p نعم. فيما يلي بعض الأمثلة التي يمكن أن تكون فيها سجلات التغيير عديمة فائدة.
+  
+  %h4#log-diffs
+    %a.anchor{ href: "#log-diffs", aria_hidden: "true" }
+    تضمين تبديلات السجلات
+
+  %p
+    
+    استخدام تضمينات 
+    %br Git
```
Comment (m-soli) [inline source/ar/1.0.0/index.html.haml:137]:
```suggestion
    Git
```

**Review by m-soli — COMMENTED** [review]
skimming through translations are accurate, styling the page to account for right to left reading of arabic and general readability is my only suggestion.

**Comment by olivierlacan** [comment]
@m-soli I believe we can do that with CSS and text-direction, right?

**Comment by RIYADHABBES** [comment]
Hi Olivier,
Hi Mark @m-soli , thank you for taking the time to review my translation,

That's true I had some trouble with mixing both Arabic and English.

I will search if there is a way in CSS or Haml to correct the style.

Thank you again

**Comment by olivierlacan** [comment]
@RIYADHABBES This is all you should need: https://github.com/olivierlacan/keep-a-changelog/blob/3aad6813caf67c3990289ea9b2c4668841e46785/source/fa-IR/1.0.0/index.html.haml#L22-L26

**Comment by RIYADHABBES** [comment]
Hello Olivier, 
Hello Mark,

That's correct I added those lines from the Farsi Translation and it did work just like magic.  
Now the text is much more readable and in a better format.
Thanks for your suggestions.

**Comment by olivierlacan** [comment]
@RIYADHABBES I'm going to merge this since despite my calls for review I've had no one offer to review this but if you can find someone to review your translation (just to be safe and catch any accidental mistakes if there are any) that would be a good idea. 

Regardless thank you *so* much for your contributions, I bet this is going to help lot of folks. 😃

File: source/ar/1.0.0/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+- gh = "https://github.com/olivierlacan/keep-a-changelog"
+- issues = "https://github.com/olivierlacan/keep-a-changelog/issues"
+- semver = "https://semver.org/"
+- shields = "https://shields.io/"
+- thechangelog = "https://changelog.com/podcast/127"
+- vandamme = "https://github.com/tech-angels/vandamme/"
+- iso = "http://www.iso.org/iso/home/standards/iso8601.htm"
+- ghr = "https://help.github.com/articles/creating-releases/"
+- gnustyle = "https://www.gnu.org/prep/standards/html_node/Style-of-Change-Logs.html#Style-of-Change-Logs"
+- gnunews = "https://www.gnu.org/prep/standards/html_node/NEWS-File.html#NEWS-File"
```
Comment (Goop-arsh) [inline source/ar/1.0.0/index.html.haml:18]:
```suggestion
- gnunews = "https://www.gnu.org/prep/standards/html_node/NEWS-File.html#NEWS-File"
```

**Review by Goop-arsh — COMMENTED** [review]
Do

**Pull request merged**